Exersises/1.h

```diff
@@ -0,0 +1,85 @@
+﻿#include <iostream>
+#include <string>
+#include <vector>   
+#include <random>
+#include <algorithm>
+#include <time.h>
+#include <boost/graph/graph_traits.hpp>
+#include <boost/graph/adjacency_list.hpp>
+#include <boost/graph/depth_first_search.hpp>
+
+using namespace std;
+using namespace boost;
+
+class disjoint_path_finder {
+
+private:
+    struct Vertex {
+        int id = -1;
+    };
+
+    typedef adjacency_list<vecS, vecS, undirectedS,
+        Vertex, no_property> Graph;
+    typedef graph_traits<Graph>::vertex_descriptor vertex_t;
+    typedef graph_traits<Graph>::edge_descriptor edge_t;
+
+    vector<int> color;
```

Comment:
The standard convention is to start names of a field in a class with _ i.e. _color, _pairs, and _g respectively. That way telling them apart from local variables is easier.